src/test/cpp/pch.h

```diff
@@ -2,10 +2,11 @@
 
 #include "catch.hpp"
 #include "winrt/base.h"
+using namespace std::literals;
 
 template<bool B> bool static_require()
 {
     return B;
 }
 
-#define STATIC_REQUIRE(...) REQUIRE(static_require<__VA_ARGS__>())
+#define STATIC_REQUIRE(...) { constexpr bool require = __VA_ARGS__; REQUIRE(static_require<require>()); }
```

Comment:
some clang limitiation?

Reply:
No, the Catch REQUIRE macro falls over with complex expressions. This prevents that from happening.